content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -26,11 +26,62 @@ scheduling and evicting Pods.
+A pod running in a Guaranteed QoS class can make use of exclusive CPUs  thanks to the [static](/docs/tasks/administer-cluster/cpu-management-policies/#static-policy) CPU management policy.
+
+When a node becomes [out of resource](/docs/tasks/administer-cluster/out-of-resource/), QoS classification helps kubelet
+to prioritize pods when the node needs to reclaim resources or when `oom_killer` needs to kill containers.
+If the `Guaranteed` pod uses memory resource lower than the requested resource, the pod has the lowest
+priority to be [evicted](/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods) by kubelet.
+The kubelet score container based on their QoS classification a `oom_score_adj` score and `oom_killer` uses `oom_score_adj` and the percentage of
+container usage of memory in the node to calculate `oom_score` for the container.
+So when the node experiences a system OOM event before kubelet to take action, the `oom_killer` scores a container
+`oom_score` and kills the container on the node with highest `oom_score` in this case Guaranteed pod has the lowest `oom_score_adj`
+so lowest chance to be killed by `oom_killer` based on [node OOM behavior](/docs/tasks/administer-cluster/out-of-resource/#node-oom-behavior).
+
+
+### Burstable
+
+For a pod to be included in the Burstable QoS, at least one container in the pod must have a resource (memory or CPU) limit or request set.
+The scheduler schedules pods running in a burstable QoS class on a node which fulfils resource requests. If pods don’t define resource limits,
+then it can use more resources than the amount requested. All Burstable pods run with [`none`](/docs/tasks/administer-cluster/cpu-management-policies/#none-policy) CPU management policy.
+
+The kubelet uses QoS classification when the kubelet needs to reclaim resources of a node due to [out of resource conditions](/docs/tasks/administer-cluster/out-of-resource/#node-conditions).
+When a node condition happens, `Burstable` pod which uses more than requested and has no resource limit
+is ranked by [Priority](/docs/concepts/configuration/pod-priority-preemption/) and its usage and they are most likely
+to be evicted first by kubelet. On the other hand, `Burstable` pods whose usage is lower than requested resource usage,
+like `Guaranteed` pods have the lowest priority to be [evicted](/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods).
+If a node experiences a system OOM event and `oom_killer` takes action before the kubelet, The `Burstable` pods have a middle `oom_score_adj`
+between Guaranteed and BestEffort class depending on memory request and the memory capacity of the node. After `BestEffort` pods Burstable pods
+have more chance to be killed by `oom_killer` based on [node OOM behavior](/docs/tasks/administer-cluster/out-of-resource/#node-oom-behavior).
+
+### BestEffort
+
+If All containers in a pod have no limits or requests on resources, the pod runs in the BestEffort QoS class.
```

Comment:
```suggestion
If all the containers in a Pod have no limits or requests on resources, the pod runs in the BestEffort QoS class.
```
(if a Pod has, say, one container with a cluster-level extended resource request, and no other limits or requests, what QoS class does it run in?)